vizro-core/docs/pages/user-guides/data.md

```diff
@@ -268,3 +265,81 @@ data_manager["slow_expire_data"].timeout = 60 * 60
 data_manager["no_expire_data"] = load_iris_data
 data_manager["no_expire_data"].timeout = 0
 ```
+
+### Parametrize data loading
+
+You can supply arguments to your dynamic data loading function that can be modified from the dashboard. For example, if you are handling big data then you might like to load on demand only one chunk of the data.
+
+To add a parameter to control a dynamic data source, do the following:
+
+- add the appropriate argument to your dynamic data function and specify a default value for the argument.
```

Comment:
Is this a sequence that needs to happen in a specific order? if so, maybe consider numbered bullets.

Reply:
Good point 👍 I added numbered bullets here because the third step depends on the second.

I also noticed that there are more places in the docs where we should consider the same. 
For example: `user-guides/run/`, `user-guides/filters/`, `user-guides/parameters/`, `user-guides/graphs` and so on. We should handle these in a separate PR.